backend/app/api/generate_problem.py

```
@@ -0,0 +1,150 @@
+                message="テーブルが作成されていません。先にテーブルを作成してください。",
+                error_code=NO_TABLES
+            )
+        
+        # 2. LLMに問題を生成させる
+        # TODO: 前回の正答率から難易度を自動調整（1-10レベル）
+        problem_info = await llm_service.generate_problem(table_schemas, prompt)
+        
+        # 3. 生成されたSQLを実行して結果を取得
+        correct_sql = problem_info["correct_sql"]
+        try:
+            expected_result = await db_service.execute_select_query(correct_sql, timeout=10)
```

Flake8 (7.2.0)

[error] line too long (92 > 79 characters)

(E501)